### DISCUSSION TITLE
warning message wrong?

### DISCUSSION THREAD

**Discussion by Dengzygx** [alpinejs/alpine#3145] [7. Bugs (archived)]
The prompt message 'Unable to initialize. Trying to load Alpine before `<body>` is available. Did you forget to add `defer` in Alpine\'s `<script>` tag?' Should the before be changed to after？

**Comment by ekwoka** [comment]
Nope, it's correct.

The statement is about what you're TRYING to do, not what you might want to TRY doing.

**Reply by Dengzygx** [reply]
thanks!
